metrics/send_metrics.go

```diff
@@ -0,0 +1,23 @@
+package metrics
+
+import (
+	"flag"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"log"
+	"net/http"
+)
+
+
+func Init() {
+	go send()
+}
+
+func send() {
+	addr := flag.String("listen-address", ":9999", "The address to listen on for HTTP requests.")
+
+	http.Handle("/metrics", promhttp.HandlerFor(prometheus.DefaultGatherer, promhttp.HandlerOpts{}))
+	promhttp.HandlerFor(prometheus.DefaultGatherer, promhttp.HandlerOpts{})
```

Comment:
what is the intent of defining default gatherer? and why do we need second line here?